### ISSUE TITLE
Error from XML parsing is not in the right part

### ISSUE THREAD

**Issue by JeromeMartinez** [MediaArea/MediaConch#191]
if I have ` error="TRUNCATED-ELEMENT:1"` in the trace in a part **not** in the FFV1 part (attribute ` parser="FFV1"` at an upper level`, the error is in the FFV1 part of the report:

```
MediaConch FFV1 Implementation Checker
TRUNCATED-ELEMENT Tests run: 5 | Results: ❌ Fail count: 5
Results: fail ❌ 
Name: Size
Offset: 51
Value context: /Segment[1]/Header[1]
Value: 742
```

error should go in its corresponding part.

**Comment by JeromeMartinez** [comment]
A method could be to filter depending of the error name (e.g. EBML part checks for errors starting by "EBML-" or "MKV-", FFV1 part checks  for errors starting by "FFV1-")

Or I add the attribute ` parser="FFV1"` for each error, maybe better for XML filtering (and not too huge, as I plan to stop to list errors after X errors)

**Comment by dericed** [comment]
Correct, currently all @e are in ffv1 part. I'll think on how to separate them in a query so that they can be sorted. Or perhaps I can sort according to the value of @e.

**Comment by JeromeMartinez** [comment]
issue I see with value of \@e is for generic tests (e.g. truncation), difficult to put somewhere especially if we have this error in both container and stream.
We need something generic for when we have multiple containers and multiple streams tests.

**Comment by dericed** [comment]
I think this is a problem with the xsl not properly contextualizing the @e. The use of @e itself is fine and scalable.

**Comment by JeromeMartinez** [comment]
change easily doable?
